labs/lab2/bigapp/Dockerfile.annotated

```diff
@@ -19,7 +19,7 @@ RUN yum -y update
 >>> cache in our intermediate cached image layer
 
 # Common Deps
-RUN yum -y install openssl procps-ng
```

Comment:
idk, the rest of the lab worked for me... so maybe the latest rhel7 image comes with it already